tests/pydantic/test_basic_types_pyd.py

```diff
@@ -0,0 +1,100 @@
+import pytest
```

Comment:
Another name complain :D. what does `_pyd` mean?

Reply:
the other test is suffixed because you cannot have the same files within multiple directories, so maybe we leave it as it is.